e2e/resources/import_structures.csv

```diff
@@ -0,0 +1,2 @@
+nom;description;téléphones;adresse 1;addresse 2;code postal;ville;site web;email;siret;email responsable;prénom responsable;nom responsable;fonction responsable;téléphones responsable
```

Comment:
question: On veut pas commiter ce fichier, si ?

Reply:
C'est un set de données pour le test du drag and drop + upload. Je suis preneur d'autres solutions.

Reply:
Je peux potentiellement essayer de réutiliser le modèle fichier que je download une step plutot mais je voulais pas créer de forte dépendences entre les steps